codalab/model/bundle_model.py

```diff
@@ -1000,15 +1010,20 @@ def transition_bundle_worker_offline(self, bundle):
                 # The user deleted the bundle or the bundle finished
                 return False
 
+            if getattr(bundle.metadata, "preemptible", False):
```

Comment:
Why can't we access the preemptible field directly and have to use `getattr`?

Reply:
I'm worried that if we do `bundle.metadata["preemptible"]`, this will break bundles that started running in the previous deploy (but continue running during the deploy) that don't yet have this metadata key yet.